person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    return true if age >= 18
+
+    false
+  end
+
+  def can_use_services?
+    return true if is_of_age? || @parent_permission
+
+    false
+  end
```

Comment:
Nice job so far👍🏼. Kindly note that it is a requirement that the `of_age` method should be a private method and you have done that fine👍🏼. but You also made the `can_use_services?` method private which shouldn't be so. since you are making use of it in the Teacher class😉. kindly ensure the `can_use_services` method is not declared as a private class😉. you can take the method above the `private` statement😉.

<img width="1007" alt="image" src="https://github.com/tsheporamantso/opp-school-library/assets/64643995/fe3b5f0c-8cfd-425f-8641-f748cb141d88">


```suggestion

  def can_use_services?
    return true if is_of_age? || @parent_permission

    false
  end

  private

  def of_age?
    return true if age >= 18

    false
  end
```

Reply:
Thank you so much will do as required.